swagger-components/schemas/customers_apis/PersonalFinancialRelationData.yaml

```
@@ -28,7 +28,7 @@ properties:
     maxItems: 12
   productsServicesTypeAdditionalInfo:
     type: string
-    pattern: '\w*\W*'
+    pattern: '[\w\W]*$'
```

@Lucas-Silva75 Faltou só o caractere ^ no inicio do pattern
```suggestion
    pattern: '^[\w\W]*$'
```

---

Boa @cleiltonoliveira , obrigado!